.kokoro/linkage-monitor.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -eo pipefail
+# Display commands being run.
+set -x
+
+cd github/gax-java/
+
+# Print out Java version
+java -version
+echo ${JOB_TYPE}
+
+mvn install -DskipTests=true -Dmaven.javadoc.skip=true -Dgcloud.download.skip=true -B -V
```

Comment:
This repo uses gradle, so I'm not sure how the linkage monitor will work here.

Reply:
I guess this is just the install command, so it should be able to run `./gradlew install`

Reply:
Thanks. I will update that.